src/background/index.js

```diff
@@ -41,7 +41,7 @@ addOwnCommands({
     const host = url && new URL(url).hostname;
     return {
       host,
-      domain: host && getDomain(host) || host,
+      domain: host && getDomain(host, { allowPrivateDomains: true }) || host,
```

Comment:
`tldts` by default does not respect the public suffix list and requires `{ allowPrivateDomains: true }` for manual opt-in.